proposals/VBnet-Coding-Guidelines.md

```diff
@@ -0,0 +1,78 @@
+# VB.net Coding Guidelines
+ 
+ Coding Conventions
+ 
+ * 80 character line limit    
+ Is the recommended maximum length of single line of code.    
+*As seen in a text editor, not in the context of the actual language specification*
+
+ * Use 4 space indentation, not tabs.
```

Comment:
💭 This is established by the .editorconfig for the repository. Duplicating items already included there seems unnecessary.

Reply:
@sharwell It doesn't harm to state it in the style guidelines though.